dali/pipeline/operators/paste/bbox_paste.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATORS_CROP_BBOX_PASTE_H_
+#define DALI_PIPELINE_OPERATORS_CROP_BBOX_PASTE_H_
+
+#include "dali/common.h"
+#include "dali/error_handling.h"
+#include "dali/pipeline/operators/common.h"
+#include "dali/pipeline/operators/operator.h"
+
+namespace dali {
+
+template <typename Backend>
+class BBoxPaste : public Operator<Backend> {
+ public:
+  explicit inline BBoxPaste(const OpSpec &spec) :
+    Operator<Backend>(spec) {
+    useLTRB = spec.GetArgument<bool>("ltrb");
+  }
+
+ protected:
+  bool useLTRB = false;
```

Comment:
very minor but according to our coding standard it should be useLTRB_

Reply:
Even further, `useLtrb_` ;)

After https://google.github.io/styleguide/cppguide.html#General_Naming_Rules:

> For some symbols, this style guide recommends names to start with a capital letter and to have a capital letter for each new word (a.k.a. "Camel Case" or "Pascal case"). When abbreviations or acronyms appear in such names, prefer to capitalize the abbreviations or acronyms as single words (i.e StartRpc(), not StartRPC()).